docsets/statsmodels/docset.json

```diff
@@ -1,17 +1,19 @@
 {
     "name": "statsmodels",
-    "version": "0.10.1",
+    "version": "0.10.2",
     "archive": "statsmodels.tgz",
     "author": {
         "name": "Angelo Varlotta",
         "link": "https://github.com/capac"
     },
-    "aliases": ["Statsmodels", "statsmodels", "Statsmodels 0.10.1", "statsmodels 0.10.1"],
+    "aliases": ["Statsmodels", "statsmodels", "Statsmodels 0.10.2", "statsmodels 0.10.2"],
 
     "specific_versions": [
         {
             "version": "0.10",
             "archive": "versions/0.10/statsmodels.tgz",
+            "version": "0.10.1",
+            "archive": "versions/0.10.1/statsmodels.tgz"
```

Comment:
`specific_versions` is supposed to be an array of dictionaries. You added the second version as extra keys within the same dictionary.

Reply:
Also, `specific_versions` need to be sorted by newest-first.